app/furniture/marketplace/routes.rb

```diff
@@ -1,7 +1,7 @@
 class Marketplace
   class Routes
     def self.append_routes(router)
-      router.resources :marketplaces, only: [:show, :edit], module: "marketplace" do
```

Comment:
Rails was complaining about no PATCH route so I opened this up.

Reply:
You should be able to add `:update` rather than removing the `only`, which exposes `create` and `destroy`